helper/ResourceClasses.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace OmekaTheme\Helper;
+
+use Laminas\View\Helper\AbstractHelper;
+
+class ResourceClasses extends AbstractHelper
+{
+
+    /**
+     * Returns a representation of resource classes in HTML.
+     *
```

Comment:
Non-blocking: As there are more protection lines than happy path lines of code in this function, would you mind adding to the function docs here as to what it protects against? `* returns and empty string if supplied with no resource, and empty resource, or a resources without ID` [I'm not sure if that's true but that's what I think is happening]